website/src/pages/index.js

```diff
@@ -13,6 +13,7 @@ const HomepageHeader = () => {
   const { siteConfig } = useDocusaurusContext();
   return (
     <header className={clsx('hero hero--primary', styles.heroBanner)}>
+      <img className={clsx(styles.graphic2)} src="static/img/mojaGraphic2.png"></img>
```

Comment:
Please add `alt`, `height` and `width` attributes to all image tags.

Reply:
Added alt, height and width attributes to all image tags.